sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -278,13 +311,25 @@ protected ServiceBusReceiverClientBuilder getReceiverBuilder(boolean useCredenti
     }
 
     protected ServiceBusSessionReceiverClientBuilder getSessionReceiverBuilder(boolean useCredentials,
-        MessagingEntityType entityType, Function<ServiceBusClientBuilder, ServiceBusClientBuilder> onBuilderCreate) {
+        MessagingEntityType entityType, Function<ServiceBusClientBuilder, ServiceBusClientBuilder> onBuilderCreate,
+        boolean sharedConnection) {
+
+        ServiceBusClientBuilder builder;
```

Comment:
can you add a getBuilder method here so we can consolidate these sections of duplicated code.